### PR TITLE
enhancement storage-network tip

### PR DESCRIPTION
**Pull request by WuJun2016**
### Summary
image

#### PR Checklist
~- Is this a multi-tenancy feature/bug?
    - [ ] Yes, the relevant RBAC changes are at:~
~- Do we need to backport changes to the [old Rancher UI](https://github.com/rancher/u), such as RKE1?
    - [ ] Yes, the relevant PR is at:~
~- Are backend engineers aware of UI changes?
    - [ ] Yes, the backend owner is:~

Fixes #

### CODE REVIEW

File: pkg/harvester/l10n/en-us.yaml [hunk 1]
```diff
@@ -848,7 +848,7 @@ harvester:
         invalid: '"Exclude list" is invalid.'
         addIp: Add Exclude IP
       warning: 'WARNING: <br/> Any change to storage-network requires shutting down all VMs before applying this setting. <br/> Users have to ensure the Cluster Network is configured and VLAN Config will cover all nodes and ensure the network connectivity is working and expected in all nodes.'
-      tip: IP Range should be IPv4 CIDR format and 4 times the number of your cluster nodes and plus the number of VM images that you will add to the cluster.
+      tip: 'IP range should be IPV4 format, the required number of IPs is calculated using a simple formula: <code> Required Number of IPs = Number of Nodes * 4 + Number of Disks * 2 + Number of Images to Download/Upload </code>. Please refer to the <a href="https://docs.harvesterhci.io/v1.2/advanced/storagenetwork#configuration-example" target="_blank">doc</a> for more details.'
```
Comment (vickyhella) [inline pkg/harvester/l10n/en-us.yaml:851]:
```suggestion
      tip: 'IP Range should be in IPV4 format. <code> Required Number of IPs = Number of Nodes * 4 + Number of Disks * 2 + Number of Images to Download/Upload </code>. See <a href="https://docs.harvesterhci.io/v1.2/advanced/storagenetwork#configuration-example" target="_blank">doc</a> for more details.'
```

File: pkg/harvester/l10n/zh-hans.yaml [hunk 2]
```diff
@@ -848,7 +848,7 @@ harvester:
         invalid: '"排除列表"无效。'
         addIp: 添加排除 IP
       warning: '警告：<br/>如果要更改存储网络，你需要在应用此设置之前关闭所有虚拟机。<br/>用户必须确保配置了集群网络，VLAN 配置能覆盖所有节点，而且所有节点的网络正常。'
-      tip: IP 范围应为 IPv4 CIDR 格式，并且是集群节点数的 4 倍加上要添加到集群的 VM 镜像数量。
+      tip: IP 范围应为 IPV4 格式, 所需 IP 数量可通过一个简单的公式计算得出：<code> 所需 IP 数量 = 节点数量 * 4 + 磁盘数量 * 2 + 要下载/上传的图像数量 </code>. 请参阅 <a href="https://docs.harvesterhci.io/v1.2/advanced/storagenetwork#configuration-example" target="_blank">文档 </a>了解更多详情.
```
Comment (vickyhella) [inline pkg/harvester/l10n/zh-hans.yaml:851]:
```suggestion
      tip: IP 范围应为 IPV4 格式。<code>所需 IP 数量 = 节点数量 * 4 + 磁盘数量 * 2 + 要下载/上传的镜像数量</code>。详见<a href="https://docs.harvesterhci.io/v1.2/advanced/storagenetwork#configuration-example" target="_blank">此文档</a>。
```

File: pkg/harvester/l10n/en-us.yaml (hunk 1 above)
Comment (vickyhella) [reply pkg/harvester/l10n/en-us.yaml:851]: Using concise sentences in the UI is more user-friendly.

**Review by vickyhella — COMMENTED** [review]
Some small suggestions on making the sentences shorter and more concise, and `image` should be `镜像`?

**Review by vickyhella — APPROVED** [review]

**Review by n313893254 — APPROVED** [review]
LGTM, Thanks!

**Comment by n313893254** [comment]
@mergify backport release-harvester-v1.2

**Comment by WuJun2016** [comment]
@mergify backport release-harvester-v1.2

**Pull request merged**